Common/include/option_structure.hpp

```diff
@@ -986,20 +988,20 @@ static const MapType<string, ENUM_ROELOWDISS> RoeLowDiss_Map = {
 enum ENUM_WALL_FUNCTIONS {
   NO_WALL_FUNCTION          = 0,   /*!< \brief No wall function treatment, integration to the wall. Default behavior. */
   STANDARD_WALL_FUNCTION    = 1,   /*!< \brief Standard wall function. */
-  ADAPTIVE_WALL_FUNCTION    = 2,   /*!< \brief Adaptive wall function. Formulation depends on y+. */
-  SCALABLE_WALL_FUNCTION    = 3,   /*!< \brief Scalable wall function. */
-  EQUILIBRIUM_WALL_MODEL    = 4,   /*!< \brief Equilibrium wall model for LES. */
-  NONEQUILIBRIUM_WALL_MODEL = 5,   /*!< \brief Non-equilibrium wall model for LES. */
-  LOGARITHMIC_WALL_MODEL    = 6    /*!< \brief Logarithmic law-of-the-wall model for LES. */
+  EQUILIBRIUM_WALL_MODEL    = 2,   /*!< \brief Equilibrium wall model for LES. */
+  LOGARITHMIC_WALL_MODEL    = 3,   /*!< \brief Reichardt's law-of-the-wall model for LES. */
+  ALGEBRAIC_WALL_MODEL      = 4,    /*!< \brief Algebraic wall model for LES. */
+  APGLL_WALL_MODEL          = 5,     /*!< \brief Adverse Pressure Gradient Wall Model for LES. */
+  TEMPLATE_WALL_MODEL       = 6     /*!< \brief Template Wall Model */
```

Comment:
Can you please add/update the `config_template.cfg `with the changes introduced in this PR.